status-server/src/public/js/client.js

```diff
@@ -191,6 +191,16 @@ function renderStatus(urlKey) {
         break;
     }
     document.getElementById("led"+idx).style.backgroundColor = color;
-    //led.fillRect(0,0,40,40);
+    Buzz(50000);
+
   }
 }
+function Buzz(m){
+  var audio = document.getElementById('buzz');
+  if (audio.paused){
```

Comment:
You have this backwards. You want to play immediately when Buzz is called, then after m seconds, pause or stop the playback. So something like: 

var audio = document.getElementById("buzz"); 
audio.load(); // should wind it back to beginning
audio.play(); 
setTimeout(function() { 
  // this function is called after the number of milliseconds passed
  audio.pause(); 
}, 150); // 100-150 seems like a good value for this sound clip